spec.html

```diff
@@ -11030,10 +11104,10 @@ <h1>ECMAScript Function Objects</h1>
             [[PrivateEnvironment]]
           </td>
           <td>
-            Environment Record
+            PrivateEnvironment Record | *null*
```

Comment:
It's probably simpler to just give scripts and modules an empty PrivateEnvironment, I guess.